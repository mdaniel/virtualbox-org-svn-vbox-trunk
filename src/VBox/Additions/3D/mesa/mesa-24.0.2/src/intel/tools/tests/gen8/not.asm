not(16)         g10<1>D         g1.2<0,1,0>D                    { align1 1H };
not(8)          g4<1>D          g8<8,8,1>D                      { align1 1Q };
