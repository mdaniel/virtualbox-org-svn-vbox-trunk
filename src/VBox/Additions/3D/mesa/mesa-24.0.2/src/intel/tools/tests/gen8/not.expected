04 00 80 00 28 0a 40 21 28 00 00 00 00 00 00 00
04 00 60 00 28 0a 80 20 00 01 8d 00 00 00 00 00
